package-lock.json

```diff
@@ -38609,7 +38609,6 @@
     "node_modules/npm-run-all": {
       "version": "4.1.5",
       "dev": true,
-      "license": "MIT",
```

Comment:
this change is also still here

Reply:
Hey @ieahleen 

My bad. For some reason I am unable to commit any suggestions in this PR. It is getting confusing. 

Let me know if there is anything else.